docs/admin/deployment.md

```diff
@@ -633,11 +745,14 @@ engines:
   fabric_iface_port: 32416
   log_mask: ERR
   log_file: /tmp/server2.log
-  scm_mount: /mnt/daos
-  scm_class: dcpm
-  scm_list: [/dev/pmem1]       # <----- updated
-  bdev_class: nvme
-  bdev_list: ["0000:da:00.0"]  # <----- updated
+  storage:
+  -
+    class: dcpm               # type of first storage tier (SCM)
+    scm_list: [/dev/pmem1]    # <----- updated
+    scm_mount: /mnt/daos      # where to mount SCM
```

Comment:
Should probably show that each engine config has a unique SCM mountpoint as well, to avoid confusion.